internal/server/middlewares.go

```diff
@@ -109,26 +111,39 @@ func requestLogger(lg *zap.Logger) middleware {
 			t := time.Now()
 			span := trace.FromContext(req.Context())
 
+			clientID, _, _ := req.BasicAuth()
+
+			wrapped := &wrappedWriter{ResponseWriter: wr, Status: http.StatusOK}
+
+			fields := []zap.Field{
+				zap.String("request_path", req.URL.Path),
+				zap.String("request_method", req.Method),
+				zap.String("request_id", req.Header.Get(headerRequestID)),
+				zap.String("client_id", clientID),
+				zap.String("trace_id", span.SpanContext().TraceID.String()),
+				zap.String("response_time", time.Since(t).String()),
+				zap.Int("status", wrapped.Status),
+			}
+
 			buf, err := io.ReadAll(req.Body)
```

Comment:
assuming only GET call that should have request body. we could rule out this body reading so we only care if method is POST or PUT.

Reply:
done